components/invitation_modal/invitation_modal_members_step/invitation_modal_members_step.jsx

```diff
@@ -177,8 +180,8 @@ class InvitationModalMembersStep extends React.PureComponent {
     };
 
     componentDidMount() {
-        if (!this.props.analytics) {
-            this.props.actions.getStandardAnalytics();
+        if (isEmpty(this.props.subscriptionStats)) {
+            this.props.actions.getSubscriptionStats();
```

Comment:
We should check here if they are on cloud license before we try to get the `subscriptionStats`

Reply:
Isn't that what the getWithCloudSubscription HOC does?

Reply:
Well `getWithCloudSubscription` does the actual get of the `subscription`, this is a new endpoint: https://github.com/mattermost/mattermost-server/pull/16786